native/connector/connect-and-monitor.sh

```diff
@@ -72,7 +93,10 @@ else
                         select(.info.props[\"media.name\"] | contains($EXCLUDED_TARGETS) | not) |
                         select(.info.props[\"media.class\"] == \"Stream/Output/Audio\" )
                     ][0].id
-                " >/dev/null || exit 0
+                " >/dev/null || (
+                    UtilLog "[connect-and-monitor.sh] [Skipped Node ID] ID: $id"
+                    exit 0
```

Comment:
There's a bug here:
Using "()" here makes this run in a subshell, and that makes the `exit 0` only exit the subshell. The idea of this exit 0 was to work as a "continue" for the while (to skip the node).

I noticed because I got the following logs:

```
[connect-and-monitor.sh] [Entering All Desktop Audio Mode] Serial: -1
[connect-and-monitor.sh] [Got Nodes IDs] IDs:
[connect-and-monitor.sh] [Got New Node ID] ID: 170
[connect-and-monitor.sh] [Skipped Node ID] ID: 170
[connect-and-monitor.sh] [Got Ports IDs] FL IDs: 173, FR IDs: 136
[connect-and-monitor.sh] [Linked Ports] 173 -> 266, 136 -> 274
```

Replacing the "()" with "{}" should work

```diff
diff --git a/native/connector/connect-and-monitor.sh b/native/connector/connect-and-monitor.sh
index c758d27..dc705dd 100755
--- a/native/connector/connect-and-monitor.sh
+++ b/native/connector/connect-and-monitor.sh
@@ -93,10 +93,10 @@ else
                         select(.info.props[\"media.name\"] | contains($EXCLUDED_TARGETS) | not) |
                         select(.info.props[\"media.class\"] == \"Stream/Output/Audio\" )
                     ][0].id
-                " >/dev/null || (
+                " >/dev/null || {
                     UtilLog "[connect-and-monitor.sh] [Skipped Node ID] ID: $id"
                     exit 0
-                )
+                }
 
                 # 1. Find the ports with node.id == $id
                 # 2. Get only the FR and FL ports
```